app/assets/javascripts/Measure.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const Code = require('./basetypes/Code');
+const Interval = require('./basetypes/Interval');
+const Quantity = require('./basetypes/Quantity');
+
+const [Integer, Array, String, Boolean, Mixed, ObjectId, Date] = [
+  mongoose.Schema.Types.Integer,
+  mongoose.Schema.Types.Array,
+  mongoose.Schema.Types.String,
+  mongoose.Schema.Types.Boolean,
+  mongoose.Schema.Types.Mixed,
+  mongoose.Schema.Types.ObjectId,
+  mongoose.Schema.Types.Date,
+];
+
+const MeasureSchema = mongoose.Schema(
+  {
+    id: String,
+    measure_id: String,
+    hqmf_id: String,
+    hqmf_set_id: String,
+    hqmf_version_number: Integer,
+    cms_id: String,
+    title: String,
+    description: String,
+    type: String,
+    category: { type: String, default: 'Uncategorized' },
+
+    episode_of_care: Boolean,
+    continuous_constiable: Boolean,
+    episode_ids: [], // not sure if this is necessary
+
+    needs_finalize: Boolean, // Bonnie-specific?
```

Comment:
Yes, this is Bonnie specific. This gets set to `true` so that after measure import, the user is asked what they want to name strats/etc. Maybe set this false by default.

Reply:
Yeah that's fine, it's definitely bonnie-only.